### ISSUE TITLE
PostgreSQL doesn't support explicit `COMMIT` and `ROLLBACK`

### ISSUE THREAD

**Issue by okbob** [darold/ora2pg#391]
These commands should be commented by default.

**Comment by darold** [comment]
I have kept those commands inside translated code intentionally, I've though that they may represented some logic in the stored procedure that should be studied during the export to see how handle this logic in the PostgreSQL context. Commenting these commands will help loading the function without error but this can hide the logic to the user.

**Comment by edelanoe** [comment]
I agree, but once the algorithmic issue has been solved in the Oracle source, the commits and rollbacks could be commented-out, perhaps provided a configuration parameter. 
For information, here is our conclusion on this topic of intermediate commit and rollback (you may put it in .conf comments): The need of a commit before the end of a function denotes that, if an error happens after this point, the changes must still be saved. 
That is: the code after the commit must be inside a BEGIN...EXCEPTION...END where the exception handler manages the error and lets the function finish.
So, basically, a COMMIT followed by a BEGIN could be commented-out, but if there is no BEGIN block, it should be left as-is, in order to keep a logic warning.
A ROLLBACK is generally used in an exception handler. In this case, it can be commented-out in Postgres. A ROLLBACK outside an exception handler should, in my opinion, be changed into an exception handler.
Thank you for any feedback.
Eric

**Comment by okbob** [comment]
I understand. The correct solution can be pretty hard, so the most correct and easy solution can be off, on switch.

**Comment by darold** [comment]
Commit 7211c7f:

    Add new COMMENT_COMMIT_ROLLBACK configuration directive. By default call to
    COMMIT/ROLLBACK are kept untouched byt Ora2Pg to force the user to review the
    logic of the function. Once it is fixed in Oracle source code or you want to
    comment this calls enable the directive.

**Comment by okbob** [comment]
I don't see this commit. Maybe it was not pushed.

**Comment by darold** [comment]
Sorry, pushed now.

**Comment by okbob** [comment]
It is working now, thank you
